jina/executors/rankers/trainer.py

```diff
@@ -0,0 +1,50 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import abc
+
+from . import BaseRanker
```

Comment:
For now, no need to add `ranker`  or anything as parameter.

For now, let's have a `RankerTrainer` as a class that trains something than then the `RankerExecutor` can understand.

But no direct link between the 2 classes